src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/IPersonGroupOperations.cs

```diff
@@ -18,23 +18,23 @@ namespace Microsoft.Azure.CognitiveServices.Vision.Face
     using System.Threading.Tasks;
 
     /// <summary>
-    /// PersonGroup operations.
+    /// PersonGroupOperations operations.
     /// </summary>
-    public partial interface IPersonGroup
+    public partial interface IPersonGroupOperations
     {
         /// <summary>
         /// Create a new person group with specified personGroupId, name and
         /// user-provided userData.
         /// </summary>
         /// <param name='personGroupId'>
-        /// User-provided personGroupId as a string.
+        /// Id referencing a particular person group.
         /// </param>
         /// <param name='name'>
-        /// Name of the face list, maximum length is 128.
+        /// Name of the face collection, maximum length is 128.
```

Comment:
Seems we need a general description for the Name. This is not for face collection.

Reply:
Done